index.js

```diff
@@ -84,8 +84,10 @@ app.get("/verify_email/:email/:token", (req, res) => {
 
 	mailService
 		.verifyEmailToken(token, email)
-		.then(email => {
-			res.send(`The email ${email} has been verified.`);
+		.then(() => {
+			res.redirect("../../views/EmailVerified.html").then(() => {
```

Comment:
once res.redirect is called, you cant send another response with res.send cause the browser isn't waiting for one. We can implement the redirect later.